lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -91,3 +91,10 @@
     margin-right: 20px;
   }
 }
+.fixed {
+  position: fixed;
+  top: 0;
+  z-index: 9999;
+  width: 100%;
+  box-shadow: 0px 3px 5px #bbb;
```

Comment:
Please remove the ``px`` value from the x-value. We do not place units of measurement on any values of 0 (see https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#numbers).